atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -959,6 +961,17 @@ private int estimateSize(List<RowResult<Value>> rangeRows) {
             }
         }
 
+        if (AtlasDbConstants.hiddenTables.contains(tableRef)) {
```

Comment:
I looked at that and believe the original code mistakenly excluded some of the hidden tables.  From what I can tell, all hidden tables are affected, not just _transactions and _namespace (really any special table that is used outside of the transaction protocol is affected).